modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './index.scss';
+
+import {useEffect, useState} from 'react';
+
+import {getPolicies} from '../../../../../../common/services';
+import {getApplications} from '../../../../../../common/services/Application';
+import {getQuotes} from '../../../../../../common/services/Quote';
+
+const QuotedSummary = ({externalReferenceCode}: any) => {
+	const [policyStartDate, setPolicyStartDate] = useState<string>('');
+	const [policyEndDate, setPolicyEndDate] = useState<string>('');
+	const [totalPremium, setTotalPremium] = useState<number>(0);
+
+	const PARAMETERS_APPLICATIONS = {
+		filter: `contains(externalReferenceCode,'${externalReferenceCode}')`,
+	};
+
+	const getPoliciesThroughApplicationERC = async () => {
+		let applicationId: number;
+		let quoteId: number;
+
+		await getApplications(PARAMETERS_APPLICATIONS).then((response: any) => {
+			applicationId = response.data.items[0].id;
+		});
+
+		await getQuotes().then((response: any) => {
+			const quoteItems = response?.data?.items;
+
+			const quoteItem = quoteItems.find(
+				(item: any) =>
+					item?.r_applicationToQuotes_c_raylifeApplicationId ===
+					applicationId
+			);
+
+			quoteId = quoteItem.id;
+		});
+
+		await getPolicies().then((response: any) => {
+			const policiesItems = response?.data?.items;
+
+			const policyItem = policiesItems.find(
+				(item: any) =>
+					item?.r_quoteToPolicies_c_raylifeQuoteId === quoteId
+			);
+
+			const startDate: string = policyItem.startDate;
+			const endDate: string = policyItem.endDate;
+			const termPremium: number = policyItem.termPremium.toFixed(2);
+
+			const formattedStartDate = new Date(startDate).toLocaleDateString(
+				'en-us',
+				{
+					day: '2-digit',
+					month: '2-digit',
+					timeZone: 'UTC',
+					year: 'numeric',
+				}
+			);
+
+			const formattedEndDate = new Date(endDate).toLocaleDateString(
+				'en-us',
+				{
+					day: '2-digit',
+					month: '2-digit',
+					timeZone: 'UTC',
+					year: 'numeric',
+				}
+			);
```

Comment:
You should try to use a generic function like:
```suggestion
			function handleDateFormatter (date) {
			const formattedDate = new Date(date).toLocaleDateString(
				'en-us',
				{
					day: '2-digit',
					month: '2-digit',
					timeZone: 'UTC',
					year: 'numeric',
				}
			);
			return formattedDate;
			}
```

Reply:
Done.